### PR TITLE
Feat/accessibility homeview

### PR DESCRIPTION
**Pull request by lkarpins**
** Resolves: #57 , #54  **

** Describe the functionality to be tested: **
- Added Prop Types where necessary 
- Fixed scroll bug
- Changed title to be uniform on all views
- Updated content to about view
- Addressed merge conflict
- Adjusted text font/spacing on all views 

** Giphy of how you feel submitting this PR: **

### CODE REVIEW

File: src/Components/ClinicCard/ClinicCard.js [hunk 1]
```diff
@@ -75,3 +76,15 @@ const ClinicCard = ({
 };
 
 export default ClinicCard;
+
```
Comment (Rallen13) [inline src/Components/ClinicCard/ClinicCard.js:79]: Great job adding proptypes to each file that needed them!

**Review by Rallen13 — APPROVED** [review]
Looks good

File: src/Components/ClinicCard/ClinicCard.js [hunk 2]
```diff
@@ -75,3 +76,15 @@ const ClinicCard = ({
...
+ClinicCard.propTypes = {
```
Comment (irmakerdem) [inline src/Components/ClinicCard/ClinicCard.js:80]: Awesome

File: src/Views/AboutView/AboutView.js [hunk 3]
```diff
@@ -1,97 +1,135 @@
...
-            informed about current laws. Abortion is essential.
-          </p>
-        </div>
-      </div>
-      <h2 data-cy="meet">Meet the Team</h2>
-      <div className="team-container">
-        <div className="photo-container">
+    <>
+      {" "}
+      {window.scroll(0, 0)}
```
Comment (irmakerdem) [inline src/Views/AboutView/AboutView.js:17]: 🪟

**Review by irmakerdem — APPROVED** [review]
Yay prop typessss

File: src/Views/AboutView/AboutView.js (hunk 3 above)
Comment (catlynbowles) [inline src/Views/AboutView/AboutView.js:17]: Was looking for this. Cool !

**Review by catlynbowles — APPROVED** [review]

**Pull request merged**